Yosemite/Yosemite/Actions/ProductCategoryAction.swift

```diff
@@ -15,6 +15,11 @@ public enum ProductCategoryAction: Action {
     ///
     case addProductCategory(siteID: Int64, name: String, parentID: Int64?, onCompletion: (Result<ProductCategory, Error>) -> Void)
 
+    /// Creates new product categories associated with a given Site ID.
+    /// `onCompletion` will be invoked when the add operation finishes. `error` will be nil if the operation succeed.
```

Comment:
Super nit: we should update the comments to match the new action:

```suggestion
    /// Creates new product categories associated with a given Site ID, category names, and an optional parent ID>
    /// `onCompletion` will be invoked when the add operation finishes.
```

Reply:
Updated in ddcb3c5cf50f66cad0dd001023586bd71a7cdb2d